core/docker/importer/openhim.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const fs = require('fs')
+const https = require('https')
+const path = require('path')
+const waitOn = require('wait-on')
+
+const OPENHIM_API_HOSTNAME =
+  process.env.OPENHIM_API_HOSTNAME || 'openhim-core'
+const OPENHIM_API_PASSWORD =
+  process.env.OPENHIM_API_PASSWORD || 'openhim-password'
+const OPENHIM_API_PORT = process.env.OPENHIM_API_PORT || 8080
+const OPENHIM_API_USERNAME =
+  process.env.OPENHIM_API_USERNAME || 'root@openhim.org'
+
+exports.importOpenHIMConfig = () => {
+  const opts = {
+    resources: [`https://${OPENHIM_API_HOSTNAME}:${OPENHIM_API_PORT}/heartbeat`],
+    delay: 1000, // initial delay in ms, default 0
+    interval: 2000, // poll interval in ms, default 250ms
+    timeout: 30000, // timeout in ms, default Infinity
+    tcpTimeout: 1000, // tcp timeout in ms, default 300ms
+    window: 1000, // stabilization time in ms, default 750ms
+    auth: {
+      user: OPENHIM_API_USERNAME,
+      pass: OPENHIM_API_PASSWORD
+    },
```

Comment:
do we need to supply the auth here for checking the heartbeat? The API doesnt require any auth on this endpoint

Reply:
No auth needed.
Fixed in https://github.com/openhie/instant/pull/6/commits/16c1169f6e260ccb42dab30f2ac0c1a7533ed998